vms/platformvm/state/camino_diff.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2022, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/components/avax"
+	"github.com/ava-labs/avalanchego/vms/platformvm/lock"
+)
+
+func (d *diff) LockedUTXOs(txID ids.ID, addresses []ids.ShortID, lockState lock.State) ([]*avax.UTXO, error) {
+	txMatch := func(lockID lock.IDs) bool {
+		switch lockState {
+		case lock.LockStateDeposited:
```

Comment:
Since we now have introduced locking of one TX == both states, we must also check for both if state is depositedBonded